tests/openapi/run.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+
+set -eu
+
+cur=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+source $cur/../_utils/test_prepare
+
+WORK_DIR=$TEST_DIR/$TEST_NAME
+
+function prepare_database() {
+	run_sql 'DROP DATABASE if exists openapi;' $MYSQL_PORT1 $MYSQL_PASSWORD1
+	run_sql 'CREATE DATABASE openapi;' $MYSQL_PORT1 $MYSQL_PASSWORD1
+
+	run_sql 'DROP DATABASE if exists openapi;' $MYSQL_PORT2 $MYSQL_PASSWORD2
+	run_sql 'CREATE DATABASE openapi;' $MYSQL_PORT2 $MYSQL_PASSWORD2
+}
+
+function test_source() {
+	echo ">>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>START TEST OPENAPI: SOURCE"
+	prepare_database
+	# create source succesfully
+	$cur/client/source.py "create_source1_success"
```

Comment:
```suggestion
	openapi_check "create_source1_success"
```

like tests/_utils/check_ticker_interval

Reply:
fixed in https://github.com/pingcap/dm/pull/2055/commits/1d68f49c3873e6c62b1e4ed7bab8832893f73034